internal/xds/bootstrap/bootstrap.go

```diff
@@ -39,14 +40,29 @@ const (
 	// DefaultWasmServerPort is the default listening port of the wasm HTTP server.
 	wasmServerPort = 18002
 
-	envoyReadinessAddress = "0.0.0.0"
-	EnvoyReadinessPort    = 19001
-	EnvoyReadinessPath    = "/ready"
+	envoyReadinessAddress     = "0.0.0.0"
+	envoyReadinessAddressIPv6 = "::"
+	EnvoyReadinessPort        = 19001
+	EnvoyReadinessPath        = "/ready"
 
 	defaultSdsTrustedCAPath   = "/sds/xds-trusted-ca.json"
 	defaultSdsCertificatePath = "/sds/xds-certificate.json"
 )
 
+func AdminAddress(family egv1a1.IPFamily) string {
+	if family == egv1a1.IPv6 {
+		return envoyAdminAddressIPv6
+	}
+	return envoyAdminAddress
+}
+
+func readinessAddress(family egv1a1.IPFamily) string {
```

Comment:
imo just the changes in this file should be enough to support pure IPv6 case as well as dual stack case, wdyt

Reply:
yes

Reply:
need changes around dynamic istener for IPv6 first/pure cluster.